packages/paypal-components/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "allowUnreachableCode": false,
+    "declaration": false,
+    "experimentalDecorators": true,
+    "strict":false,
```

Comment:
Can our components really not be built with strict?

Reply:
ts-config was copied from bedrock-components, which should also be strict 🤔 updated!

Reply:
resolved 👍